python/ray/data/dataset.py

```diff
@@ -646,12 +646,6 @@ def add_column(
             ray_remote_args: Additional resource requirements to request from
                 ray (e.g., num_gpus=1 to request GPUs for the map tasks).
         """
-        compute = get_compute_strategy(
```

Comment:
`map_batches` calls `get_compute_strategy` inside it. So we don't need to call `get_compute_strategy` here.